src/test/java/org/folio/dcb/controller/BaseIT.java

```diff
@@ -38,6 +39,8 @@
 public class BaseIT {
   @Autowired
   protected MockMvc mockMvc;
+//  @MockBean
```

Comment:
I think we can remove this commented line if it is not needed

Reply:
done